server/mailet/mock-smtp-server/src/main/java/org/apache/james/mock/smtp/server/model/Mail.java

```diff
@@ -41,6 +41,9 @@
 
 @JsonDeserialize(builder = Mail.Builder.class)
 public class Mail {
+
+    public static final List<String> smtpParametersWithoutValues = List.of("REQUIRETLS", "SMTPUTF8");
```

Comment:
Can we avoid being hardcoding a list of values here?

If we just say 'if it do not contains a `=`' then we associate `""` then no specific whitelist is needed?

Reply:
Yes, we can get rid of the whitelist. But it seems to me that the mock server operation would look more plausible if it only allowed certain parameters without values. If we need to, we can decide how to pass the list as arguments or properties or whatever. But if it really doesn't make sense, I'll just leave it possible for any parameter to have an empty value.

Reply:
My concern would be keep adding value and handling those two in a different fashion.

If needed we could go from `Map<String, String>` as a datastructure to something like `Map<String, Optional<String>>`?